src/ui/pages/config/ToolbarPageConfig.js

```diff
@@ -51,7 +56,19 @@ class ToolbarPageConfig extends Component {
           </span>
         </span>
         <div className="panel-group">
-          <div className="ToolbarPageConfig__drawer-pf-container" >
+          <div
+            className={classScrollContainer.join(' ')}
+            onMouseEnter={(
+              () => {
+                document.body.style.overflow = 'hidden';
```

Comment:
please use classes instead of setting DOM properties though JS

Reply:
done. please review @gidesan